tests/containers/base_image_test.py

```diff
@@ -233,3 +257,18 @@ def encode_python_function_execution_command_interpreter(python: str, function:
         print({name}({parameters}));""")
     int_cmd = [python, "-c", program]
     return int_cmd
+
+import docker.errors
+import docker.models.images
+
+def is_rstudio_image(my_image: str) -> bool:
```

Comment:
would be nice to avoid duplicating

https://github.com/opendatahub-io/notebooks/blob/02e6ef3aa8a862dd7206d59a7250f21c83af2578/tests/containers/conftest.py#L92-L99

but I don't insist on it here

Reply:
adjusted